sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_alternative_document_input_async.py

```diff
@@ -37,11 +37,11 @@ async def alternative_document_input(self):
         text_analytics_client = TextAnalyticsClient(endpoint=endpoint, credential=AzureKeyCredential(key))
 
         documents = [
-            {"id": "0", "language": "en", "text": "I had the best day of my life."},
-            {"id": "1", "language": "en",
+            {"id": "0", "country_hint": "US", "text": "I had the best day of my life. I decided to go sky-diving and it made me appreciate my whole life so much more. I developed a deep-connection with my instructor as well."},
+            {"id": "1", "country_hint": "GB",
```

Comment:
just curious, why in some cases the whole object is in one line, and in some it is divided? I can't find the pattern

Reply:
For dicts idk if it would work to have it on multiple lines, I'm not sure how it would render. Otherwise, I try to make it as document-like as possible. In cases where I have to print the document, I can't make it as document-like as possible, since that format introduces whitespace that looks weird when printed